src/core/emergency-action.c

```diff
@@ -31,7 +31,7 @@
 static void log_and_status(Manager *m, const char *message, const char *reason) {
         log_warning("%s: %s", message, reason);
         manager_status_printf(m, STATUS_TYPE_EMERGENCY,
-                              ANSI_HIGHLIGHT_RED " !!  " ANSI_NORMAL,
+                              ANSI_HIGHLIGHT_RED "  !!  " ANSI_NORMAL,
```

Comment:
This change is not described in the commit message :(, so I can't be sure about the motivation, but it seems wrong: the formatting was asymmetrical because of the total required number of characters.

Reply:
Discussion is always welcome, it stops some of my mistakes from getting merged.

Notice the `error_message` one is six characters, `"!!!!!!"`.  My memory also said that `DEP` gets printed off-center, so the number of characters should be odd.  (It would also seem an "interesting" choice to go for an off-center `!!`, if you could have a centered `!!!` instead).

I'll double-check the behaviour and call out the change somehow.

Reply:
Ah, now I see (and remember): the total space is 6 characters ("  OK  ", "FAILED", etc.). So it was two exclamation marks to have it centered. I'm not sure how we ended up with two **asymmetrical** exclamation marks. So yeah, your change seems to be correct.